woostarter/patterns/about-centered-founder-bio.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Title: Centered founder bio/story
+ * Slug: woostarter/founder-bio
+ * Categories: text, about, media
+ * Keywords: about, text, icons, bio, story
+ * Description: A section with a heading, image, paragraph and button.
+ *
+
+?>
+<!-- wp:group {"metadata":{"categories":["intro"],"name":"Centered founder bio/story","patternName":"a8c/intro-two-column-with-image-and-content"},"align":"full","className":"alignfull is-style-default","style":{"spacing":{"margin":{"top":"0","bottom":"0"},"padding":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained","justifyContent":"center"}} -->
+<div class="wp-block-group alignfull is-style-default" style="margin-top:0;margin-bottom:0;padding-top:0;padding-bottom:0"><!-- wp:spacer {"height":"var:preset|spacing|70"} -->
+<div style="height:var(--wp--preset--spacing--70)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|20","padding":{"top":"0","bottom":"0"}}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
+<div class="wp-block-group" style="padding-top:0;padding-bottom:0"><!-- wp:paragraph {"align":"center","style":{"typography":{"textTransform":"uppercase","letterSpacing":"0.1em","fontStyle":"normal","fontWeight":"600","fontSize":"12px"}}} -->
+<p class="has-text-align-center" style="font-size:12px;font-style:normal;font-weight:600;letter-spacing:0.1em;text-transform:uppercase">Meet the founder</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading {"textAlign":"center","level":3,"fontSize":"xx-large"} -->
+<h3 class="wp-block-heading has-text-align-center has-xx-large-font-size">Luiza Saito</h3>
+<!-- /wp:heading --></div>
+<!-- /wp:group -->
+
+<!-- wp:spacer {"height":"var:preset|spacing|20"} -->
+<div style="height:var(--wp--preset--spacing--20)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:image {"id":189,"aspectRatio":"1","scale":"cover","sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full"><img src="https://kiri.mystagingwebsite.com/wp-content/uploads/2024/11/tempImagepYO0T3.jpg" alt="A soft-focus black and white portrait of a woman sitting outdoors with a large dog, surrounded by trees." class="wp-image-189" style="aspect-ratio:1;object-fit:cover"/></figure>
```

Comment:
This `src` attribute needs to be updated to `<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/tempImagepYO0T3.jpg` before merging.

We may want to rename the file to something more descriptive but not sure if that matters _quite_ as much.

Reply:
Thanks! I submitted the changes